sensor_stack/sparton/src/sparton.py

```diff
@@ -241,10 +246,71 @@ def getData():
     allData.data[12] = temp
     
     return allData
-    
- 
-if __name__ == '__main__':
-    """    
+
+def getOrientation():
+
+    global roll
+    global pitch
+    global yaw
+
+    quaternion = 
+
+    return quaternion
+
+def getOrientationCovariance():
+
+    # global roll
+    # global pitch
+    # global yaw
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+def getAngularVelocityCovariance():
+
+    # global gx
+    # global gy
+    # global gz
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+
+def getLinearAccelerationCovariance():
+
+    # global ax
+    # global ay
+    # global az
+
+    cov_mat = [0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0]
+
+    return cov_mat
+
+def getQuaternion():
```

Comment:
@icyflame : Found some useful stuff regarding the inbuilt `tf` transformations here : http://answers.ros.org/question/69754/quaternion-transformations-in-python/